erigon-lib/common/collections.go

```diff
@@ -20,12 +20,6 @@ import (
 	"math/rand"
 )
 
-func SliceReverse[T any](s []T) {
```

Comment:
Why remove?

Reply:
replaced with go's built in std `slices` package - `slices.Reverse`